rust-runtime/aws-smithy-http-server/examples/Cargo.toml

```diff
@@ -5,3 +5,6 @@ members = [
     "pokemon_service_sdk",
     "pokemon_service_client"
 ]
+
+[profile.release]
+lto = true
```

Comment:
Hmm does this have any effect? I thought enabling LTO in the binary would also enable it in all the crates it depends on.

Reply:
It does enable LTO on all dependent crates resulting in a longer compilation time. We can remove this, it will probably lower the amount of requests / sec we can do by a little. I don't have a strong opinion here.